src/tests/ftest/io/nvme_io.py

```diff
@@ -93,12 +89,13 @@ def test_nvme_io(self):
         sizes.IOR will be run for all Object type supported. Purpose is to
         verify pool size (SCM and NVMe) for IOR file.
         This test is running multiple IOR on same server start instance.
-        :avocado: tags=nvme,nvme_io,large
+        :avocado: tags=hw,nvme,large
```

Comment:
Every test needs a unique global tag so that it can be targeted individually if desired.

Does this test absolutely require a _large_ cluster to run?

Reply:
You should align your changes here with what is happening in #574 to reduce the amount of conflict that patch will have when it lands.

Reply:
I will update tags